intg-androidtv/driver.py

```diff
@@ -361,6 +361,19 @@ async def main():
         await config.devices.migrate()
     # and register them as available devices.
     for device in config.devices.all():
+        # Migration of certificate/key files with identifier in name
+        _android_tv = tv.AndroidTv(api.config_dir_path, device.address, device.name, device.id)
+        if not os.path.exists(_android_tv.certfile):
+            current_certfile = api.config_dir_path + "/androidtv_remote_cert.pem"
+            current_keyfile = api.config_dir_path + "/androidtv_remote_key.pem"
```

Comment:
Use `os.path.join` to create file paths.

Reply:
OK

Reply:
Done